src/mixin-tester.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div class="d-none" />
+</template>
+
+<script lang="ts">
+import { Component, Mixins } from 'vue-property-decorator'
+import { DateMixin } from '@/mixins/date-mixin'
+
+/** Special component used only for testing mixins. */
+@Component({})
+export default class MixinTester extends Mixins(DateMixin) {}
```

Comment:
Shouldn't this live in TESTS? or is that not possible?

Reply:
21 unit tests seems fairly low as well lol.

Reply:
Good call. Fixed.